MekHQ/src/mekhq/campaign/io/CampaignXmlParser.java

```diff
@@ -772,7 +775,7 @@ private static void processLanceNodes(Campaign retVal, Node wn) {
             }
 
             if (!wn2.getNodeName().equalsIgnoreCase("lance")) {
-                // Error condition of sorts!
+                // Error condition of [sorts]!
```

Comment:
What is this change in relation to?

Reply:
Don't know where it came from, maybe the IDE, I did not edit this, I've reverted it in recent changes.